src/main/java/io/confluent/connect/jdbc/util/DateTimeUtils.java

```diff
@@ -77,6 +79,24 @@ public static String formatTimestamp(Date date, TimeZone timeZone) {
     }).format(date);
   }
 
+  private static Long convertToEpochMicros(Timestamp t) {
+    Long epochMillis = TimeUnit.SECONDS.toMicros(t.getTime() / MILLISECONDS_PER_SECOND);
+    Long microsInSecond = TimeUnit.MICROSECONDS.toMicros(t.getNanos() / MILLISECONDS_PER_SECOND);
+    return epochMillis + microsInSecond;
```

Comment:
nit: Better readability.
```suggestion
    Long microsComponent = t.getNanos() / NANOSECONDS_PER_MICROSECOND;
    return epochMillis + microsComponent;
```

Where NANOSECONDS_PER_MICROSECONDS is a constant defined as:
`static final long NANOSECONDS_PER_MICROSECOND = TimeUnit.MICROSECONDS.toNanos(1);`